packages/grafana-ui/src/schema.ts

```diff
@@ -7,7 +7,7 @@ export {
   GraphDrawStyle as DrawStyle,
   // All exports
   AxisPlacement,
-  PointVisibility,
+  VisibilityMode as PointVisibility,
```

Comment:
note -- old names are exported here, so there should not be any external effects to this change